apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -171,6 +171,25 @@ ConversationView.prototype = {
     );
   },
 
+  enterEditMode: function() {
+    this.showOptions();
+    this.menuAccessors.selectAppMenuOption('Select Messages');
+  },
+
+  exitEditMode: function() {
+    this.back();
+  },
+
+  isInEditMode: function() {
```

Comment:
Not all checkboxes are visible at any given time. One needs to scroll to ensure that.